src/components/AccountContainer.js

```diff
@@ -6,56 +6,48 @@ class AccountContainer extends Component {
 
   constructor() {
     super()
-
-    // we have provided this default state for you,
-    // use this to get the functionality working
-    // and then replace the default transactions with a call to the API
-
     this.state = {
-      searchTerm: '',
-      transactions: [
-        {
-          id: 1,
-          posted_at: "2017-02-28 11:00:00",
-          description: "Leather Pants, Gap co.",
-          category: "Fashion",
-          amount: -20000
-        },
-        {
-          id: 2,
-          posted_at: "2017-02-29 10:30:00",
-          description: "Paycheck from Bob's Burgers",
-          category: "Income",
-          amount: 100000
-        },
-        {
-          id: 3,
-          posted_at: "2017-05-24 10:53:00",
-          description: "'Pair Programming Illuminated' by Laurie Williams and Robert Kessler",
-          category: "Book",
-          amount: 1498
-        },
-        {
-          id: 4,
-          posted_at: "2017-05-24 08:52:00",
-          description: "Medium Iced Cold Brew, Gregory's Coffee",
-          category: "Coffee",
-          amount: 365
-        }
-      ]
+      transactions: [],
+      searchTerm: ''
     }
+    this.handleChange = this.handleChange.bind(this)
   }
 
-  handleChange(event) {
-    // your code here
+  componentDidMount(){
+  fetch('https://boiling-brook-94902.herokuapp.com/transactions')
+    .then( res => res.json() )
+    .then( data => this.setState({ transactions: data}) )
+}
+
+handleChange(event){
+  const ans = []
+  const arr = this.state.transactions
+  const st = event.target.value
+  if (st.length > 0){
+  for (var i in arr){
+    if (arr[i].category.indexOf(st) > -1 || arr[i].description.indexOf(st) > -1){
+    ans.push(arr[i])
+    }
   }
+  this.setState({
```

Comment:
We discussed this in the review and you mentioned this in your comment - you don't want to overwrite the list of transactions because then you won't have them.